src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -106,7 +107,7 @@ void AddHintsIfAppropriate(SyntaxNode node)
                     {
                         result.Add(new InlineHint(
                             new TextSpan(position, 0),
-                            ImmutableArray.Create(new TaggedText(TextTags.Text, parameter.Name + ": ")),
+                            ImmutableArray.Create(new TaggedText(TextTags.Text, HintText(parameter.Name))),
```

Comment:
intresting.  this used to add colon-space.  now it seems like it will only add colon for the C# case.  is that intentional?  can you have a picture for both C# and VB after this?

Reply:
I meant to keep the space. So C# looks as it normally does and VB looks like this-
![image](https://user-images.githubusercontent.com/40616383/141359005-d9fb251f-4604-47a1-a6ae-1da16de04ef3.png)